Src/Enums.cs

```diff
@@ -6,6 +6,8 @@ public enum KtaneModuleType
         Regular,
         [KtaneFilterOption("Needy module", 'y')]
         Needy,
+        [KtaneFilterOption("Hodable", 'H')]
```

Comment:
Little spelling mistake:
```suggestion
        [KtaneFilterOption("Holdable", 'H')]
```